apps/question-bank/src/test/resources/features/page/staticelement/AddHyperLinkStaticElementToPage.feature

```diff
@@ -0,0 +1,30 @@
+@create_hyperlink
+Feature: Create Hyperlink 
+
+    Background:
+        Given I have a page
+        And the page has a tab
+        And the page has a section in the 1st tab
+        And the page has a sub-section named "sub-section" in the 1st section
+
+    Scenario: I can create a hyperlink static element
+        Given I am an admin user
+        When I send a hyperlink request with "<label>" and "<link>"
+        Then a hyperlink is created with "<label>" and "<link>"
+        Examples:
+            | label      |
+            | google     | 
+            | yahoo      |
+            | link       |
+            | google.com |
+            | yahoo.com  |
```

Comment:
I think you intended to do this:
```
Examples:
            | label      |  link|
            | google     |  google.com |
            | yahoo      | yahoo.com|
```

Reply:
Functionally does it still work and take in the correct example strings or does it now mix match the parameters?